<launch>
    <arg name="robot_name" default="small_scout_1" />



    <include file="$(find planning)/launch/planner.launch"/>
    <include file="$(find perception)/launch/object_detection.launch"/>
    <include file="$(find maploc)/launch/mapping.launch"/>
    <include file="$(find operations)/launch/navigation.launch">
        <arg name="robot_name" value="$(arg robot_name)" />
    </include>
    

    <!--group ns="/capricorn/$(arg robot_name)">
        <node pkg="operations" type="start_scout_sm" name="scout_1_sm" args="$(arg robot_name)" output="screen" />
        <node pkg="operations" type="resource_localiser" name="scout_1_localiser" args="$(arg robot_name)" output="screen" />
    </group-->
    <node pkg="operations" type="start_scout_sm" name="scout_1_sm" args="$(arg robot_name)" output="screen" />
    <node pkg="operations" type="navigation_client" name="scout_1_client" args="$(arg robot_name)" output="screen" />
</launch>